### PR TITLE
Add CLASS to check 96

### PR DESCRIPTION
**Pull request by ibnbr**
#110

### CODE REVIEW

**Comment by larshp** [comment]
closes #110

File: src/checks/zcl_aoc_check_96.clas.abap [hunk 1]
```diff
@@ -48,20 +55,52 @@ CLASS ZCL_AOC_CHECK_96 IMPLEMENTATION.
...
+    DATA: lv_program_name TYPE programm,
+          lt_mtdkeys      TYPE seo_cpdkeys.
+    FIELD-SYMBOLS <ls_mtdkey> LIKE LINE OF lt_mtdkeys.
+
+    IF object_type = 'CLAS'.
+      SELECT clsname cmpname
+        FROM vseocompdf
+        INTO TABLE lt_mtdkeys
+        WHERE clsname = object_name
+          AND cmptype = '1'.
```
Comment (larshp) [inline src/checks/zcl_aoc_check_96.clas.abap]: please add field VERSION in this select, we dont want to check inactive stuff

File: src/checks/zcl_aoc_check_96.clas.abap [hunk 2]
```diff
@@ -48,20 +55,52 @@ CLASS ZCL_AOC_CHECK_96 IMPLEMENTATION.
...
+          EXCEPTIONS
+            class_not_existing  = 1
+            method_not_existing = 2
+            OTHERS              = 3 ).                    "#EC CI_SUBRC
+
+        IF is_edt_lock( lv_program_name ) = abap_true.
+          inform( p_sub_obj_type = c_type_include
+                  p_sub_obj_name = lv_program_name
+                  p_kind    = mv_errty
+                  p_test    = myname
```
Comment (larshp) [inline src/checks/zcl_aoc_check_96.clas.abap]: as far as I remember `p_test    = myname` can be omitted? also another occurrence below

Comment (ibnbr) [reply src/checks/zcl_aoc_check_96.clas.abap]: I can't omit.
image

File: src/checks/zcl_aoc_check_96.clas.abap (hunk 1 above)
Comment (ibnbr) [reply src/checks/zcl_aoc_check_96.clas.abap]: Done.

**Pull request merged**